llarp/router/outbound_session_maker.hpp

```diff
@@ -52,11 +52,20 @@ namespace llarp
     util::StatusObject
     ExtractStatus() const override;
 
+    bool
+    ShouldConnectTo(const RouterID &router) const override
+        LOCKS_EXCLUDED(_mutex);
+
     void
     Init(ILinkManager *linkManager, I_RCLookupHandler *rcLookup,
          std::shared_ptr< Logic > logic, llarp_nodedb *nodedb,
          std::shared_ptr< llarp::thread::ThreadPool > threadpool);
 
+    /// always maintain this many connections to other routers
+    size_t minConnectedRouters = 4;
+    /// hard upperbound limit on the number of router to router connections
+    size_t maxConnectedRouters = 6;
```

Comment:
... and likewise here (but also 6 here vs 5 earlier; perhaps a constant for this instead?)

Reply:
overriden by limits later

Reply:
Actually I think this is just the comment that is wrong -- this looks like the client limits rather than router limits.

Reply:
yeah the comments are wrong, i could initialize with zero but that'd be weird. the values are overrided by limits later in the code.